qa/suites/rgw/verify/tasks/s3tests.yaml

```diff
@@ -1,5 +1,5 @@
 tasks:
 - s3tests:
     client.0:
-      force-branch: ceph-master
+      force-branch: tethology_s3select_apr19
```

Comment:
oops! this part shouldn't have merged

Reply:
is there anything on this `tethology_s3select_apr19` branch that should be merged into `ceph-master`?

Reply:
no. that should not be merged into ceph-master